src/site/content/en/learn/css/selectors/index.md

```diff
@@ -0,0 +1,486 @@
+have `<div>` child elements.
+
+#### Next sibling combinator
+
+You can look for an element that immediately follows another element
+by using a `+` character in your selector.
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'JjEPzwB'
+} %}
+
+To add space between stacked elements,
+use the next sibling combinator to add space
+*only* if an element is a **next sibling** of a child element of `.top`.
+
+You could add margin to all child elements of `.top`,
+using the following selector:
+
+```css
+.top * {
+	margin-top: 1em;
+}
+```
+
+The problem with this is that because you're selecting every child element of `.top`,
+this rule potentially creates extra, unnecessary space.
+The **next sibling combinator**,
+mixed with a **universal selector** enables you to not only control what elements get space,
+but also apply space to **any element**.
+This provides you with some long-term flexibility,
+regardless of what HTML elements appear in `.top`.
+
+#### Subsequent- sibling combinator
+
+A subsequent combinator is very similar to a next sibling selector.
+Instead of a `+` character however,
+use a `~` character.
+How this differs is that an element just has to follow another element with the same parent,
+rather than being the next element with the same parent.
+
+<figure class="w-figure">
+  {% Codepen {
+    user: 'web-dot-dev',
+    id: 'ZELzPPX'
+  } %}
+  <figcaption class="w-figcaption">Use a subsequent selector along with a `:checked` pseudo class to create a pure CSS switch element.</figcaption>
+</figure>
+
+This subsequent combinator provides a little less rigidity,
+which is useful in contexts like the above sample,
+where we change the colour of a custom switch when its associated checkbox has the `:checked` state.
+
+#### Child combinator
+
+A child combinator (also known as direct descendant)
+allows you more control over the recursion that comes with combinator selectors.
+By using the `>` character, you limit the combinator selector to apply **only** to direct children.
+
+Consider the previous, next sibling selector example.
+The space is added to each **next sibling**,
+but if one of those elements also has **next sibling elements** as children,
+it can result in undesirable, extra spacing.
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'ExZYMJL'
+} %}
+
+To alleviate this problem,
+change the **next sibling selector** to incorporate a child combinator: `> * + *`.
+The rule will now **only** apply to direct children of `.top`.
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'dyNbrEr'
+} %}
+
+### Compound selectors
+
+You can combine selectors to increase specificity and readability.
+For example, to target `<a>` elements,
+that also have a class of `.my-class`, write the following:
+
+```css
+a.my-class {
+	color: red;
+}
+```
+
+This wouldn't apply a red colour to all links
+and it would also only apply the red colour to `.my-class` **if** it was on an `<a>` element.
```

Comment:
"and it would also only apply the red colour to `.my-class` **if** it was on an `<a>` element." --> "and it would also only apply the red color to any `<a>` elements (links) with a class of `.my-class`.